keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -299,14 +301,16 @@ spec:
 +  - name: example-com-server-tls-trust-anchors
 +    projected:
 +      sources:
-+      - pemTrustAnchors:
-+          clusterTrustBundleName: example.com--server-tls-live
++      - clusterTrustBundle:
++          signerName: example.com/server-tls
++          labelSelector:
++            kubernetes.io/cluster-trust-bundle-version: live
```

Comment:
Remember to document `kubernetes.io/cluster-trust-bundle-version` in https://kubernetes.io/docs/reference/labels-annotations-taints/